processor/README.md

```diff
@@ -235,15 +235,20 @@ examples on using the processor.
 The batch processor accepts spans and places them into batches grouped by node
 and resource. Batching helps better compress the data and reduce the number of
 outgoing connections required to transmit the data. This processor supports
-both size and time based batching. Please refer to
-[config.go](batchprocessor/config.go) for the config spec.
+both size and time based batching.
+
+It is highly recommended to configure the batch processor on every collector.
+The batch processor should be the second processor defined in the pipeline
+(immediately after the memory_limiter processor).
+
+Please refer to [config.go](batchprocessor/config.go) for the config spec.
 
 The following configuration options can be modified:
-- `num_tickers` (default = 4): the number of tickers that loop over batch buckets
-- `remove_after_ticks` (default = 10): the number of ticks passed without a span arriving for a node at which time batcher is deleted
-- `send_batch_size` (default = 8192): size after which a batch will be sent regardless of time
-- `tick_time` (default = 1s): interval in which the tickers tick
-- `timeout` (default = 1s): time duration after which a batch will be sent regardless of size
+- `num_tickers` (default = 4): Number of tickers that loop over batch buckets
+- `remove_after_ticks` (default = 10): Number of ticks passed without a span arriving for a node at which time batcher is deleted
+- `send_batch_size` (default = 8192): Size after which a batch will be sent regardless of time
```

Comment:
I think it is worth to mention that this is the number of spans, since it can be interpreted as size by users.

Reply:
Makes sense